perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -654,7 +654,7 @@ sub response_templates {
 #     Note:   this only makes sense when called on a problem that has been sent!
 sub can_display_external_id {
     my $self = shift;
-    if ($self->external_id && $self->send_method_used && $self->bodies_str =~ /2237/) {
+    if ($self->external_id && $self->send_method_used && $self->bodies_str =~ /(2237|2550)/) {
```

Comment:
Not sure that matters – this check only needs to be in the `UK.pm` cobrand, if I understand it right; it can then still check hard-coded body IDs, that's fine, but if it could call a function in `$c->cobrand` then those hard-coded IDs would only be in a place that they make sense in.